frontend/src/components/RoadmapGraphSidebar.module.scss

```diff
@@ -30,9 +30,18 @@
 .metrics {
   @extend .layout-row;
   gap: 10px;
-  height: 120px;
+  height: max-content;
   align-items: center;
   justify-content: center;
+  div {
+    white-space: normal !important;
+    height: calc(100% - 8px);
```

Comment:
This is not a likely scenario, but the "dash" can become misshapen if the value has spaces.
To avoid this, the border on MetricsSummary .dash could be changed to `border-top: 1px solid $COLOR_BLACK40;`
![image](https://user-images.githubusercontent.com/80758782/170068986-91eeb308-3756-4141-8a79-652c4986ea65.png)